src/frontend/src/flows/pin/pinInfo.json

```diff
@@ -0,0 +1,18 @@
+{
+  "en": {
+    "continue_and_set_pin": "Continue and set pin",
+    "cancel": "Cancel",
+    "create_temporary_key_form_pin": "Create a temporary key from a PIN",
+    "set_memorable_pin": "Set a memorable PIN and securely store a temporary key in your browser cache",
```

Comment:
not related to your PR, just as a feedback, don't understand what's "memorable PIN". Also "securely store a temporary key in your browser cache" sounds like an oxymoron.

Reply:
Yeah, I will probably need to tweak the text later on during testing. This is just taken as is from the figmas.

Reply:
Just my two cents, not related to the PR